actioncable/test/test_helper.rb

```diff
@@ -65,6 +66,8 @@ class ActionCable::TestCase < ActiveSupport::TestCase
     include ConcurrentRubyConcurrencyHelpers
   end
 
+  include EnvHelpers
```

Comment:
Why not just define these methods in here like `ConcurrentRubyConcurrencyHelpers`?